src/metadata/exiv2_handler.cc

```diff
@@ -137,29 +137,29 @@ void Exiv2Handler::fillMetadata(const std::shared_ptr<CdsObject>& item)
         // if there are any auxilary tags that the user wants - add them
         const auto aux = config->getArrayOption(CFG_IMPORT_LIBOPTS_EXIV2_AUXDATA_TAGS_LIST);
         if (!aux.empty()) {
-            std::string value;
+            std::string val;
             std::string auxtag;
 
             for (auto&& j : aux) {
-                value.clear();
+                val.clear();
                 auxtag = j;
                 log_debug("auxtag: {} ", auxtag.c_str());
                 if (auxtag.substr(0, 4) == "Exif") {
-                    const auto md = exifData.findKey(Exiv2::ExifKey(auxtag));
+                    md = exifData.findKey(Exiv2::ExifKey(auxtag));
                     if (md != exifData.end())
-                        value = md->toString();
+                        val = md->toString();
                 } else if (auxtag.substr(0, 3) == "Xmp") {
-                    const auto md = xmpData.findKey(Exiv2::XmpKey(auxtag));
+                    auto md = xmpData.findKey(Exiv2::XmpKey(auxtag));
```

Comment:
You don't need `auto` here either.

Reply:
Yeah you do. xmpData is a difference type than ExifData.

Reply:
Then choose another name like `xmpMd`.